nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/nifi.properties

```diff
@@ -111,9 +111,11 @@ nifi.components.status.repository.buffer.size=${nifi.components.status.repositor
 nifi.components.status.snapshot.frequency=${nifi.components.status.snapshot.frequency}
 
 # Site to Site properties
-nifi.remote.input.socket.host=
-nifi.remote.input.socket.port=
+nifi.remote.input.host=
 nifi.remote.input.secure=true
+nifi.remote.input.socket.port=
+nifi.remote.input.http.enabled=true
```

Comment:
`nifi.remote.input.http.enabled` and `nifi.remote.input.secure` are set to true by default, but https port, keystore and truststore are not configured. With the default nifi.properties file, user will see following exception at start up, which is not a good UX. Can we change default `nifi.remote.input.secure` to `false` ?

```
Caused by: java.lang.RuntimeException: Remote input HTTPS is enabled but nifi.web.https.port is not specified.
        at org.apache.nifi.util.NiFiProperties.getRemoteInputHttpPort(NiFiProperties.java:439) ~[nifi-properties-1.0.0-SNAPSHOT.jar:1.0.0-SNAPSHOT]
```